modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/StructuredContentDTOConverter.java

```diff
@@ -184,6 +184,7 @@ public StructuredContent toDTO(
 				renderedContents = _toRenderedContents(
 					ddmStructure, dtoConverterContext, journalArticle);
 				siteId = GroupUtil.getSiteId(group);
+				structuredContentFolderId = journalArticle.getFolderId();
```

Comment:
It would be great if we test that the endpoint is returning the id properly. For that, please, have a look at the tests in cluded in the 
 `StructuredContentResourceTest` class that overrrides a base clase generated by the REST Builder. There are some tests that are already testing some endpoints, and there are some methods that are overriden to indicate which fields must be validated, for example `getAdditionalAssertFieldNames`. Maybe it is necessary to include the new field in some methods to validate the change. Please, have a look at it

Reply:
Updated PR with requested changes, thank you!

Reply:
In the end, I've skipped adding `structuredContentFolderId` field to `getAdditionalAssertFieldNames` list because it breaks all tests related to PUT and PATCH methods. Basically PUT/PATCH generated test methods compare updated `structuredContent` with the `randomStructuredContent` object generated in the test. They are different because some `structuredContent` fields are readOnly and their values are skipped when submitted to the PUT endpoint while present in the `randomStructuredContent` object. If we want to add `structuredContentFolderId` field to `getAdditionalAssertFieldNames`